Program.cs

```diff
@@ -118,7 +120,11 @@ static void Main(string[] args)
                 }
 
                 var snapshotTipHeader = snapshotTipDigest.Value.Header;
-                var json = JsonSerializer.Serialize(snapshotTipHeader);
+                var jObj = new JObject();
+                jObj = JObject.FromObject(snapshotTipHeader);
```

Comment:
The `new JObject()` allocation seems useless. In my thought, it is enough with `JObject.FromObject()`.
```suggestion
                var jObj = JObject.FromObject(snapshotTipHeader);
```